ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/management-api-gnmi.j2

```diff
@@ -17,3 +17,24 @@ management api gnmi
    provider eos-native
 {%     endif %}
 {% endif %}
+{% if management_api_gnmi.transport is arista.avd.defined %}
```

Comment:
please enclose all the new config inside the `if` block started in line 2 and ending in line 19. So you only evaluate everything if the outer `if` is true.

Reply:
done